packages/graphql/src/graphql/ops/FQuestionDataWithoutSolutions.graphql

```diff
@@ -4,13 +4,15 @@ fragment QuestionDataWithoutSolutions on QuestionInstance {
     questionId
     name
     type
-    displayMode
     content
     explanation
     pointsMultiplier
 
     ... on ChoicesQuestionData {
       options {
+        displayMode
+        hasSampleSolution
```

Comment:
remove hasSampleSolution and hasAnswerFeedbacks

Reply:
done